zapier/zapier-platform
# feat(schema) - Add lock config

Project: https://coda.io/d/Team-Developer-Platform_di0MgBhlCWf/Technical-Decision-CLI-Capability-Locking_su-UM#_luO3f
Ticket: https://zapierorg.atlassian.net/browse/PDE-4238

Adds the optional `lock` config to the `BasicOperationConfigSchema`. 

---

### Testing Locally
1. Created a CLI app. 
2. Added `lock` config that looks like this
 ```
  lock: {
      key: 'fetch_lyrics',
      timeout: 50,
      scope: ['account'],
    },
```
3. Run local `zapier` validate (using the schema in this PR)
4. See that it passes the local validation
```
Validating project locally
No structural errors found during validation routine.
This project is structurally sound!
```
(Note fails the API check bc the schema is not updated in the platform version)
